<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>350</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>175</horstretch>
     <verstretch>175</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Light</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }
    QPushButton:pressed {
    background-color:rgb(180,180,180);
    }</string>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }</string>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Пароль</string>
   </property>
   <property name="toolTipDuration">
    <number>3</number>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
    }</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Serif Light</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Serif Light</family>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Light</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
}

QCheckBox::indicator:unchecked {
     background-color:rgb(180,180,180);
 }

QCheckBox::indicator:checked {
     background-color:rgb(66,66,66);
 }
</string>
   </property>
   <property name="text">
    <string>Запомнить меня</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Light</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator{
    border: none;
    background-color:rgb(230,230,230);
    border-radius: 5px;
}

QCheckBox::indicator:unchecked {
     background-color:rgb(180,180,180);
 }

QCheckBox::indicator:checked {
     background-color:rgb(66,66,66);
 }
</string>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>230</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
